guard/src/commands/validate/tf.rs

```diff
@@ -276,14 +259,14 @@ fn single_line(
                         re: &UnaryComparison<'_>,
                         prefix: &str,
                     ) -> crate::rules::Result<usize> {
-                        let width = "PropertyPath".len() + 4;
+                        let _width = "PropertyPath".len() + 4;
```

Comment:
Okay, so this variable is being used and still has a prefix? 

Is [this](https://stackoverflow.com/a/48361729) the correct explanation?

Reply:
It's not being used, I should have deleted it...For some reason it was defined there and then again [7 lines later](https://github.com/aws-cloudformation/cloudformation-guard/blob/main/guard/src/commands/validate/tf.rs#L286)